docker/pytorch-aarch64/build.sh

```diff
@@ -87,6 +88,7 @@ nproc_build=
 onednn=
 target="native"
 clean_build=
+xla=0
```

Comment:
I would follow the approach we have for onednn and set `xla=`.

Reply:
ok